tekton/cache-fetch.yaml

```diff
@@ -38,6 +38,11 @@ spec:
         The path where to find the google credentials. If left empty, it is ignored.
       type: string
       default: ""
+    - name: docker-config
+      description: |
+        The path where to find the docker config.json file. If left empty, it is ignored.
+      type: string
+      default: "/workspace/docker"
```

Comment:
I wonder if the name should be *just* `docker` for the workspace or something else.

Reply:
> I wonder if the name should be _just_ `docker` for the workspace or something else.

what about .docker ?

Reply:
I was more thinking of `docker-config` or something that makes it a bit more clear what we are "waiting" for.